components/org.wso2.carbon.identity.handler.event.account.lock/src/main/java/org/wso2/carbon/identity/handler/event/account/lock/AccountLockHandler.java

```diff
@@ -734,4 +741,40 @@ private void setUserClaim(String claimName, String claimValue, UserStoreManager
             throw new AccountLockException("Error while setting user claim value :" + username, e);
         }
     }
+
+
+    /**
+     * To create an audit message based on provided parameters.
+     *
+     * @param action     Activity
+     * @param target     Target affected by this activity.
+     * @param dataObject Information passed along with the request.
+     * @param result     Result value.
+     */
+    private static void createAuditMessage(String action, String target, JSONObject dataObject, String result) {
+
+        String loggedInUser = PrivilegedCarbonContext.getThreadLocalCarbonContext().getUsername();
+        if (StringUtils.isBlank(loggedInUser)) {
+            loggedInUser = AuditConstants.REGISTRY_SYSTEM_USERNAME;
+        }
+        String tenantDomain = PrivilegedCarbonContext.getThreadLocalCarbonContext().getTenantDomain();
+        loggedInUser = UserCoreUtil.addTenantDomainToEntry(loggedInUser, tenantDomain);
+        AUDIT_LOG.info(String.format(AuditConstants.AUDIT_MESSAGE, loggedInUser, action, target, dataObject, result));
+    }
+
+    private void auditAccountLock(String action, String target, String userStoreDomainName, boolean isAdminInitiated,
```

Comment:
Not strictly necessary but it is better if you can add a doc comment for this method stating the functionality.

Reply:
Fixed with https://github.com/wso2-extensions/identity-event-handler-account-lock/pull/73